test/fixtures/composectl_cmds.go

```diff
@@ -0,0 +1,193 @@
+package fixtures
```

Comment:
I think you'll want this file name to end with `_test.go` to make sure people don't try and use this as part of you API.

Reply:
I assume the package name, i.e. `fixtures` indicates that this is not API.

Reply:
In the end, a client make use of these fixtures in their tests.

Reply:
i think a client could for their tests if it ends in _test.go. But making _test.go makes this explicit and makes sure it doesn't contribute to the final binary size.

Reply:
It doesn't contribute to the final binary size.

It makes sense to add suffix `_test.go` to the files containing test fixtures and test code if they are located in the same directory/package as the files of the code under the test. In this case compiler excludes `*_test.go` files from binary even if they are part of the code package.

In this particular case, fixtures are placed into the dedicated directory/package and are not compiled unless explicitly imported.